feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/ClaimsFlowActivity.kt

```diff
@@ -26,8 +28,15 @@ class ClaimsFlowActivity : ComponentActivity() {
     val token = authenticationTokenService.authenticationToken
     requireNotNull(token)
     val odysseyUrl = intent.getStringExtra(ODYSSEY_URL_KEY) ?: error("ODYSSEY_URL_KEY needs to be passed in")
+    val itemType = intent.getStringExtra(EXTRA_ITEM_TYPE)
+
     val locale = languageService.getLocale().toString()
 
+    val scopeValues = ScopeValues()
+    if (itemType != null) {
+      scopeValues.setValue(InitialDataScopeValueKey, mapOf("itemType" to itemType))
+    }
```

Comment:
So if itemType is null, we simply send an empty ScopeValue object right? Which I guess is perfectly fine if there's nothing to send anyway.

Reply:
Exactly!